src/Runner.Discord/Responders/PullTheLeverResponder.cs

```diff
@@ -74,24 +74,26 @@ private async Task<bool> CheckWithinRateLimit(IUser user)
 
         public async Task ProcessMessage(IMessage message, CancellationToken token)
         {
+            string storedMessage = message.Content.ToLower();
```

Comment:
`messageContent`? nothing is stored

Reply:
Hating on my naming conventions? :P

Reply:
Also this does not fix the command from being trigger from say, "lever the pull" but who cares, it's all in the bot channel

Reply:
that's fine - asking about the naming as reading this back doesn't make no sense -- stored implies it was stored somewhere

Reply:
Lemme clone again, this was a in-browser edit